deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/template/QuteTemplatingEngineAdapter.java

```diff
@@ -0,0 +1,73 @@
+package io.quarkiverse.openapi.generator.deployment.template;
+
+import java.io.IOException;
+import java.util.Map;
+
+import org.openapitools.codegen.api.AbstractTemplatingEngineAdapter;
+import org.openapitools.codegen.api.TemplatingExecutor;
+
+import io.quarkus.qute.Engine;
+import io.quarkus.qute.ReflectionValueResolver;
+import io.quarkus.qute.Template;
+
+public class QuteTemplatingEngineAdapter extends AbstractTemplatingEngineAdapter {
+
+    public static final String IDENTIFIER = "qute";
+    public static final String[] INCLUDE_TEMPLATES = {
+            "additionalEnumTypeAnnotations.qute",
+            "additionalModelTypeAnnotations.qute",
+            "beanValidation.qute",
+            "beanValidationCore.qute",
+            "beanValidationHeaderParams.qute",
+            "bodyParams.qute",
+            "enumClass.qute",
+            "enumOuterClass.qute",
+            "formParams.qute",
+            "headerParams.qute",
+            "pathParams.qute",
+            "pojo.qute",
+            "queryParams.qute"
+    };
+    public final Engine engine;
+
+    public QuteTemplatingEngineAdapter() {
+        this.engine = Engine.builder()
+                .addDefaults()
+                .addValueResolver(new ReflectionValueResolver())
+                .removeStandaloneLines(true)
+                .strictRendering(false)
+                .build();
```

Comment:
When running in Quarkus, the Engine is available as a CDI managed object. Maybe lookup first before attempting to create a new one?

Reply:
The CDI engine is unavailable at this stage (CodeGenProvider): I suppose it's running even before the processor. So that's why I'm manually creating it.